Debugger plugin: potential cppcheck false positive: uninitvar

cppcheck 2.2 complains about many `ifdef`s in the Debugger plugin:
```
make[5]: Leaving directory '/build/geany-plugins-1.36-1+20201013git06a38f4/debugger/src'
tpage.c:154:62: warning: Uninitialized variable: widgets [uninitvar]
   gtk_container_remove(GTK_CONTAINER(gtk_widget_get_parent(*widgets[i])), *widgets[i]);
                                                             ^
nofile:0:0: warning: Too many #ifdef configurations - cppcheck only checks 12 configurations. Use --force to check all configurations. For more details, use --enable=information. [toomanyconfigs]

make[4]: *** [Makefile:1686: check-cppcheck] Error 2
```

When we remove GTK2 code, this might resolve it.

What should we do until then?
- use `--force`?
- refactor the code?
- ignore?

This breaks the nightly builds for Debian Sid: https://www.geany.org/download/nightly-builds/.

--- a/debugger/src/tpage.c
+++ b/debugger/src/tpage.c
@@ -142,19 +142,16 @@ void tpage_pack_widgets(gboolean tabbed)
 	GList *children = gtk_container_get_children(GTK_CONTAINER(tab_target));
 	if (children)
 	{
-		int i;
-
 		oldroot = (GtkWidget*)children->data;
-		
+
 		/* unparent widgets */
-		i = 0;
-		while (widgets[i])
+		for (int i = 0; widgets[i]; i++)
 		{
-			g_object_ref(*widgets[i]);
-			gtk_container_remove(GTK_CONTAINER(gtk_widget_get_parent(*widgets[i])), *widgets[i]);
-			i++;
+			GtkWidget **widget_ref = widgets[i];
+			g_object_ref(*widget_ref);
+			gtk_container_remove(GTK_CONTAINER(gtk_widget_get_parent(*widget_ref)), *widget_ref);
 		}
-		
+
 		g_list_free(children);
 	}
 	
@@ -265,11 +262,9 @@ void tpage_pack_widgets(gboolean tabbed)
 
 	if (oldroot)
 	{
-		int i = 0;
-		while (widgets[i])
+		for (int i = 0; widgets[i]; i++)
 		{
 			g_object_unref(*widgets[i]);
-			i++;
 		}
 		gtk_container_remove(GTK_CONTAINER(tab_target), oldroot);
 	}

